web/app/scripts/views/record/list-controller.js

```diff
@@ -33,12 +37,27 @@
                 });
         }
 
-        function loadRecords() {
+        /*
+         * Loads a page of records from the API
+         * @param {int} offset Optional offset value, relative to current offset, used
+         *                     for pulling paginated results. May be positive or negative.
+         * @return {promise} Promise to load records
+         */
+        function loadRecords(offset) {
             /* jshint camelcase: false */
-            return Records.get({ record_type: $stateParams.rtuuid })
+            var params = { record_type: $stateParams.rtuuid };
             /* jshint camelcase: true */
+
+            if (offset) {
+                ctl.currentOffset += offset;
+                if (ctl.currentOffset) {
+                    params.offset = ctl.currentOffset;
+                }
+            }
+
+            return Records.get(params)
```

Comment:
Yeah, it is an interesting conundrum: we have the next/previous URLs readily available, but it isn't obvious how to easily use them. I searched a bit, but haven't been able to find a good solution.